### ISSUE TITLE
camelCase options

### ISSUE THREAD

**Issue by mourner** [mapbox/tile-cover#68]
`min_zoom` and `max_zoom` options break all JS conventions by not being camelCase, which is pretty confusing. I think we should eventually break compat and rename them (with a major version bump).

**Comment by morganherlocker** [comment]
Agree. I think this style may have been a carry over from s2, but I cannot remember.
